model_compression_toolkit/target_platform_capabilities/schema/v1.py

```diff
@@ -638,38 +619,37 @@ class TargetPlatformCapabilities(BaseModel):
         SCHEMA_VERSION (int): Version of the schema for the Target Platform Model.
     """
     default_qco: QuantizationConfigOptions
-    operator_set: Optional[Tuple[OperatorsSet, ...]]
-    fusing_patterns: Optional[Tuple[Fusing, ...]]
-    tpc_minor_version: Optional[int]
-    tpc_patch_version: Optional[int]
-    tpc_platform_type: Optional[str]
+    operator_set: Optional[Tuple[OperatorsSet, ...]] = None
```

Comment:
Are these default values required?

we prefer no to set default values to TPC attributes

Reply:
It is required

Reply:
Ok I think there is no risk in adding this default value in this case